packages/node_modules/@ciscospark/redux-module-media/src/actions.js

```diff
@@ -70,6 +87,7 @@ export function bindCallActions(dispatch, call) {
     localMediaStreamUrl: call.localMediaStreamUrl
   };
   call.on(`connected`, () => dispatch(callConnected(call)));
```

Comment:
I'm guessing we can change these to `call.once` methods since they shouldn't be called with a `connected` or `disconnected` more than once. That will remove the watcher after it happens too. @ianwremmel right?